templates/previewDebugUrl.twig

```diff
@@ -0,0 +1,7 @@
+<br>
+<div class="preview-debug-url-div">
+    <input type="url" id="previewDebugUrl" class="browser-default" placeholder="https://yourdebugsite.com" value="{{ debugSiteUrl|e('html_attr') }}">
```

Comment:
Let's maybe use a different domain just in case someone is registering that one and does something not good with it :)

In https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/Installation/tests/UI/expected-screenshots/Installation_setup_website.png we use `https://example.org`

Not sure if it makes sense to set the placeholder as the main url if there is any for the current site. Maybe people would get confused and think it's already pre-filled?

Reply:
Changed it to https://example.org and I think we should keep that only as it could be confusing if we keep the entered URL as the placeholder